airflow/operators/dagrun_operator.py

```diff
@@ -38,7 +39,8 @@ class TriggerDagRunLink(BaseOperatorLink):
     name = 'Triggered DAG'
 
     def get_link(self, operator, dttm):
-        return f"/graph?dag_id={operator.trigger_dag_id}&root=&execution_date={quote(dttm.isoformat())}"
+        view = conf.get('webserver', 'dag_default_view').lower()
+        return f"/{view}?dag_id={operator.trigger_dag_id}&root=&execution_date={quote(dttm.isoformat())}"
```

Comment:
Should we add base_url here?

Reply:
I'm afraid now it won't work properly if you run Airflow with [``base_url``](https://airflow.readthedocs.io/en/latest/configurations-ref.html#base-url) or [``proxy-fix-prefix``]https://airflow.readthedocs.io/en/latest/configurations-ref.html#proxy-fix-x-prefix] configured.

Reply:
When Airflow is installed at `http://localhost:8080/my-company/airflow,` users will redirect to `http://localhost:8080/graph` instead of `http://localhost:8080/my-company/airflow/graph`.

Reply:
When using `base_url` in the url and using breeze I'm redirected to `localhost:8080` instead of `0.0.0.0:28080`. So I think we don't have to use it it in url.

When using `url_for` then the link doesn't work at all

Reply:
Looks like it remains an open issue for v2 when prefixing `/base/` in `base_url`. Is there any issue open here for tracking?